Tests/SentryTests/Integrations/SessionReplay/SentrySessionReplayIntegrationTests.swift

```diff
@@ -576,7 +576,68 @@ class SentrySessionReplayIntegrationTests: XCTestCase {
         
         XCTAssertEqual(window.subviews.count, 0, "Mask preview should not appear in production")
     }
-    
+
+    func testMoveCurrentReplay_whenLastFileExistsWithoutCurrent_shouldBeRemoved() throws {
+        // -- Arrange --
+        startSDK(sessionSampleRate: 0, errorSampleRate: 1)
+        let sut = try getSut()
+
+        let replayFolder = sut.replayDirectory()
+        try FileManager.default.createDirectory(atPath: replayFolder.path, withIntermediateDirectories: true)
+
+        let currentReplayPath = replayFolder.appendingPathComponent("replay.current")
+        // Cleanup stale files from previous tests
+        if FileManager.default.fileExists(atPath: currentReplayPath.path) {
+            try FileManager.default.removeItem(atPath: currentReplayPath.path)
+        }
```

Comment:
`m`: Shouldn't we move that to the top of the test method, and/or maybe put this into `defer` to ensure it's always cleaned up?